Update README.md

Marta109 opened the pull request:
Hello!
This pull request is invalid, please check the code using this pull request link, thanks).
https://github.com/Marta109/word-counter-Marta-Hayrapetyan/pull/2

Marta109 committed:
Update README.md

sergkarapetian commented:
Hello Marta. Sorry, but your code is not working correctly. It does not meet the "sentence count" requirement because it counts numbers instead of sentences.

Also, it counts symbols as letters, so it doesn't meet the "letter count" requirement either. But don't get discouraged, just try another way that you think might work.

Marta109 commented:
Hello!
This pull request is invalid, please check the code using this pull request link, thanks).
https://github.com/Marta109/word-counter-Marta-Hayrapetyan/pull/2

Marta109 commented:
Hello Sergey. I changed the code and also added to the README.md  file a description of what the calculator thinks and how. Please read and check again, thanks).
